server/types.d.ts

```diff
@@ -0,0 +1,10 @@
+interface Note {
```

Comment:
I despise duplicated code, don't we already have types?

Reply:
I've temporarily added it for ease for work.